enclave-manager/server/server.go

```diff
@@ -227,26 +227,23 @@ func (c *WebServer) RunStarlarkPackage(ctx context.Context, req *connect.Request
 		Msg: req.Msg.RunStarlarkPackageArgs,
 	}
 
-	apicStream, err := (*apiContainerServiceClient).RunStarlarkPackage(ctx, serviceRequest)
-	ctxWithCancel, cancel := context.WithCancel(ctx)
-	logs := getRuntimeLogsWhenCreatingEnclave(cancel, apicStream)
-	for {
-		select {
-		case <-ctxWithCancel.Done():
-			err := apicStream.Close()
-			close(logs)
-			if err != nil {
-				return stacktrace.Propagate(err, "Error occurred after closing the stream")
-			}
-			return nil
-		case resp := <-logs:
-			errWhileSending := str.Send(resp)
-			if errWhileSending != nil {
-				logrus.Errorf("error occurred: %+v", errWhileSending)
-				return stacktrace.Propagate(errWhileSending, "Error occurred while sending streams")
-			}
+	apiClient, err := (*apiContainerServiceClient).RunStarlarkPackage(ctx, serviceRequest)
+
+	for apiClient.Receive() {
+		resp := apiClient.Msg()
+		errWhileSending := str.Send(resp)
+		if errWhileSending != nil {
+			logrus.Errorf("Error occurred: %+v", errWhileSending)
```

Comment:
I think it's worth making the error messages here more descriptive so that it's easier for an engineer debugging to locate where it's coming from in our codebase. Maybe by adding some context about where the error surfaced (eg. `while streaming from enclave manager...`) Same for the `stacktrace` error here and below.